pkg/api/v1beta3/conversion.go

```diff
@@ -3227,3 +2746,474 @@ func init() {
+	if in.Selector != nil {
+		out.Selector = make(map[string]string)
+		for key, val := range in.Selector {
+			out.Selector[key] = val
+		}
+	}
+	if err := s.Convert(&in.TemplateRef, &out.TemplateRef, 0); err != nil {
+		return err
+	}
+	if err := s.Convert(&in.Template, &out.Template, 0); err != nil {
+		return err
+	}
+	return nil
+}
+
+func convert_api_ReplicationControllerSpec_To_v1beta3_ReplicationControllerSpec(in *newer.ReplicationControllerSpec, out *ReplicationControllerSpec, s conversion.Scope) error {
+	out.Replicas = in.Replicas
+	if in.Selector != nil {
+		out.Selector = make(map[string]string)
+		for key, val := range in.Selector {
+			out.Selector[key] = val
+		}
+	}
+	if err := s.Convert(&in.TemplateRef, &out.TemplateRef, 0); err != nil {
+		return err
+	}
+	if err := s.Convert(&in.Template, &out.Template, 0); err != nil {
+		return err
+	}
+	return nil
+}
+
+func convert_v1beta3_ReplicationControllerStatus_To_api_ReplicationControllerStatus(in *ReplicationControllerStatus, out *newer.ReplicationControllerStatus, s conversion.Scope) error {
+	out.Replicas = in.Replicas
+	return nil
+}
+
+func convert_api_ReplicationControllerStatus_To_v1beta3_ReplicationControllerStatus(in *newer.ReplicationControllerStatus, out *ReplicationControllerStatus, s conversion.Scope) error {
+	out.Replicas = in.Replicas
+	return nil
+}
+
+func convert_v1beta3_Container_To_api_Container(in *Container, out *newer.Container, s conversion.Scope) error {
+	out.Name = in.Name
+	out.Image = in.Image
+	if in.Command != nil {
+		out.Command = make([]string, len(in.Command))
+		for i := range in.Command {
+			out.Command[i] = in.Command[i]
+		}
+	}
+	if in.Args != nil {
+		out.Args = make([]string, len(in.Args))
+		for i := range in.Args {
+			out.Args[i] = in.Args[i]
+		}
+	}
+	out.WorkingDir = in.WorkingDir
+	if in.Ports != nil {
+		out.Ports = make([]newer.ContainerPort, len(in.Ports))
+		for i := range in.Ports {
+			if err := s.Convert(&in.Ports[i], &out.Ports[i], 0); err != nil {
+				return err
+			}
+		}
+	}
+	if in.Env != nil {
+		out.Env = make([]newer.EnvVar, len(in.Env))
+		for i := range in.Env {
+			if err := s.Convert(&in.Env[i], &out.Env[i], 0); err != nil {
+				return err
+			}
+		}
+	}
+	if err := s.Convert(&in.Resources, &out.Resources, 0); err != nil {
+		return err
+	}
+	if in.VolumeMounts != nil {
+		out.VolumeMounts = make([]newer.VolumeMount, len(in.VolumeMounts))
+		for i := range in.VolumeMounts {
+			if err := s.Convert(&in.VolumeMounts[i], &out.VolumeMounts[i], 0); err != nil {
+				return err
+			}
+		}
+	}
+	if err := s.Convert(&in.LivenessProbe, &out.LivenessProbe, 0); err != nil {
+		return err
+	}
+	if err := s.Convert(&in.ReadinessProbe, &out.ReadinessProbe, 0); err != nil {
+		return err
+	}
+	if err := s.Convert(&in.Lifecycle, &out.Lifecycle, 0); err != nil {
+		return err
+	}
+	out.TerminationMessagePath = in.TerminationMessagePath
+	out.ImagePullPolicy = newer.PullPolicy(in.ImagePullPolicy)
+	if in.SecurityContext != nil {
```

Comment:
TL;DR this if statement and subsequent `Convert` call consitute 50% of the real content of this diff.